account/file_store.go

```diff
@@ -91,6 +95,23 @@ func NewWalletData() *WalletData {
 	}
 }
 
+func (this *WalletData) Dump() *WalletData {
```

Comment:
question: is this Copy() ?

Reply:
Yes, deep copy.

Reply:
It does not matter actually. This function is used with reencrypt.